lib/solidus_importer/process_row.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  ##
+  # This class process a single row of an import running the configured
+  # processor in chain.
+  class ProcessRow
+    def initialize(importer, row)
+      @importer = importer
+      @row = row
+      validate_context
```

Comment:
(as above for ProcessImport) validate_context here doesn't seem to actually validate the context, but rather, to validate the ProcessRow instance, what about naming it `validate!` (the bang would signal it's gonna raise an exception if it's not valid)